src/NLog/Targets/ConsoleTarget.cs

```diff
@@ -99,6 +99,12 @@ public Encoding Encoding
 #endif
 
         /// <summary>
+        /// Gets or sets a value indicating whether to auto-check if the console is available
+        /// </summary>
```

Comment:
Maybe we should tell in the comments when this is ckecked and what the concenquenes are. What do you think?